--- configure
+++ configure
@@ -2116,6 +2116,7 @@
 fi
 
 echo "$ac_t""$wi_cv_shared_libgcc" 1>&6
+if false; then
 	if test "$wi_cv_shared_libgcc" != no ; then
 		
 	if test "x$ac_cv_prog_gcc" = "xyes" ; then
@@ -2198,6 +2199,7 @@
 	fi
 
 	fi
+fi
 
 	add_O0="no"
 	if [ "$NOOPTCFLAGS" = "" ] ; then
@@ -3284,6 +3286,7 @@
 fi
 
 echo "$ac_t""$wi_cv_shared_libgcc" 1>&6
+if false; then
 	if test "$wi_cv_shared_libgcc" != no ; then
 		
 	if test "x$ac_cv_prog_gcc" = "xyes" ; then
@@ -3366,6 +3369,7 @@
 	fi
 
 	fi
+fi
 
 	add_O0="no"
 	if [ "$NOOPTCFLAGS" = "" ] ; then
